backend/git_real/tests/factories.py

```diff
@@ -73,16 +57,3 @@ class Meta:
     title = "title"
     body = "body"
     state = PullRequest.OPEN
-
-
-class PullRequestReviewFactory(DjangoModelFactory):
-    class Meta:
-        model = PullRequestReview
```

Comment:
This is a little strange. Why are you removing this code? 

Always remember to do a self-review